docs/source/algorithms/ConvertToMD-v1.rst

```diff
@@ -281,6 +288,35 @@ This example produces 3-dimensional dataset, with a temperature axis.
 
    Resulting MD workspace contains 605880 events and 3 dimensions
 
+**Example - Convert to Q-space, indexed version:**
+
+The code iteslef works but disabled from doc tests as takes too long to complete. User should provide its own
+event nexus file instead of **TOPAZ_3132_event.nxs** used within this example. The original **TOPAZ_3132_event.nxs**
+file is available in `Mantid system tests repository <https://github.com/mantidproject/systemtests/tree/master/Data/TOPAZ_3132_event.nxs>`_.
+
+
+.. code-block:: python
+   :linenos:
+
+   #.. testcode:: ExConvertToMD|Q|
+
+   # load test workspace
+   TOPAZ_3132_event = Load(Filename=r'TOPAZ_3132_event.nxs', ConverterType='Indexed')
+
+   # build peak workspace necessary for IntegrateEllipsoids algorithm to work
+   TOPAZ_3132_md = ConvertToMD(InputWorkspace=TOPAZ_3132_event,QDimensions='Q3D',dEAnalysisMode='Elastic',Q3DFrames='Q_sample',LorentzCorrection='1',\
+   MinValues='-25,-25,-25',MaxValues='25,25,25',SplitInto='2',SplitThreshold='50',MaxRecursionDepth='13',MinRecursionDepth='7')
```

Comment:
I think you said that the performance is better with a larger splitting factor, e.g., 4. Maybe that should be used in the example?

Have you considered recommending a certain split factor here, which gives good performance?

Reply:
Yes: bigger -> better. I can't give any recommendations because it depends on the task and what kind of hierarchy you want to have finally.

Reply:
I think you can and should give a recommendation, even if it is just "prefer 4 or larger, unless you require 2".